driver-core/src/test/java/com/datastax/driver/core/DataTypeTest.java

```diff
@@ -331,7 +331,7 @@ public void should_not_return_v4_types_in_all_primitive_types_with_v3() {
         }
     }
 
-    @Test(groups = "unit")
+    @Test(groups = "unit", enabled = false)
     public void should_return_same_elements_with_all_primitive_types_using_latest_protocol_version() {
         assertThat(DataType.allPrimitiveTypes(ProtocolVersion.NEWEST_SUPPORTED)).isEqualTo(DataType.allPrimitiveTypes());
```

Comment:
`should_not_return_v4_types_in_all_primitive_types_with_v3` (right above) could be deleted too.